### ISSUE TITLE
show percentage change over time

### ISSUE THREAD

**Issue by simonmichael** [simonmichael/hledger#1397]
`balance assets -M -%` (eg) will show each account's percentage of the total on a given day. But there's no easy way to see the percentage change in an account's balance (or value) over time. The `roi` command is the nearest thing. `register` doesn't support -%/--percent.

**Comment by lestephane** [comment]
Indeed, this would be useful to quickly determine which monthly expenses are abnormaly higher than the previous month when doing a monthly eyeball check.
